erpnext/stock/report/stock_balance/stock_balance.py

```diff
@@ -215,66 +233,75 @@ def get_columns(filters):
 	return columns
 
 
-def get_conditions(filters):
-	conditions = ""
-	if not filters.get("from_date"):
-		frappe.throw(_("'From Date' is required"))
```

Comment:
From Date is not validated ? Its required for opening stock or sumn right ?

Reply:
Yes, it's used. But it doesn't _have to be mandatory_, felt this was unnecessary validation. Removing it means opening stock computation won't be done.

Reply:
Oh wait, clientside JS prevents it from loading 🤦

Reply:
> Yes, it's used. But it doesn't have to be mandatory, felt this was unnecessary validation. Removing it means opening stock computation won't be done, which is fine?

Apart from JS, its also unexpected for users using this report via API for years (we have some)